README.md

```diff
@@ -123,3 +126,18 @@ Duplicate trip:
 [+] Trip is being processed...
 [+] Trip duplicated
 ```
+
```

Comment:
I'm not ok to use a Makefile for a pure ruby project. The philosophy here should be to use a gemspec file to make easier the installing/uninstalling steps. So I did, I'm going to push it after some tests.

Moreover, this tools doesn't need to be run as root (or a powerful user), and if an user install it ('sudo make install' or 'su - root' then 'make install') I'm not sure which system's account the "binary" is going to use when called. There is so different Linux/Unix operating system. The gemspec method is a better way.

Please remove all Makefile's part in the readme.md and the makefile as well.